src/components/molecules/VenueInfoEvents/VenueInfoEvents.tsx

```diff
@@ -4,38 +4,48 @@ import "./VenueInfoEvents.scss";
 import { AnyVenue } from "types/Firestore";
 import { WithId } from "utils/id";
 import { venuePlayaPreviewUrl } from "utils/url";
+import { EventDisplay } from "../EventDisplay/EventDisplay";
+import "../../../../src/components/molecules/EventDisplay/EventDisplay.scss";
```

Comment:
a dramatic journey to the file! Can be written as `../EventDisplay/EventDisplay.scss`